src/modules/NetHwDetection.rb

```diff
@@ -197,8 +198,8 @@ def DuplicateIP(ip)
       # floods logs.
       return false if ip.nil? || ip.empty?
 
-      command = "LC_ALL=C ip link show|grep BROADCAST|grep -v NOARP|cut -d: -f2"
-      exe = Convert.to_map(SCR.Execute(path(".target.bash_output"), command))
+      command = "LC_ALL=C /sbin/ip link show | /usr/bin/grep BROADCAST | /usr/bin/grep -v NOARP | /usr/bin/cut -d: -f2"
```

Comment:
`ip` is everywhere but `/usr/sbin/ip` is the real file

Reply:
on my machine both exists, but I agree we should prefer /usr prefixed variant.

Reply:
in fact ```/bin/ip``` and ```/sbin/ip``` are just symlinks ...